README.md

```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
**Highlights**

It's so nice that you included a screenshot of the project. It helps to make the repo professional. Keep up the great work!